marklogic-spark-connector/src/main/java/com/marklogic/spark/writer/DocBuilder.java

```diff
@@ -310,17 +308,14 @@ private List<DocumentWriteOperation> buildChunkDocuments(DocumentInputs inputs,
             // If there's an extracted doc, we want to use that as the source document so that the user has the option
             // of adding chunks to it.
             DocumentWriteOperation sourceDocument = extractedTextDocument != null ? extractedTextDocument : mainDocument;
-            Iterator<DocumentWriteOperation> iterator = chunkAssembler.assembleChunks(sourceDocument, inputs.getChunks(), inputs.getClassifications());
+            Iterator<DocumentWriteOperation> iterator = chunkAssembler.assembleChunks(
+                sourceDocument,
+                inputs.getChunks(),
+                inputs.getClassifications(),
+                inputs.getEmbeddings());
```

Comment:
This makes so much more sense. I'm trying to figure out why I would've done embeddings in a different manner but can't think of a reason.